cli/azd/.vscode/cspell-azd-dictionary.txt

```diff
@@ -2,9 +2,13 @@ AADSTS
 ABRT
 aiomysql
 aiopg
+aistudio
+alexwolf
```

Comment:
some of this is already in the config. Like: https://github.com/Azure/azure-dev/blob/main/cli/azd/.vscode/cspell.yaml#L39

Some cases are covered per file only.